packages/yoga/src/Dropdown/web/Dropdown.jsx

```diff
@@ -141,6 +141,7 @@ const Button = styled.button`
     width: 100%;
     padding-right: ${dropdown.button.padding.right}px;
     padding-top: ${dropdown.button.padding.top}px;
+    padding-bottom: ${dropdown.button.padding.top}px;
```

Comment:
I'm happy for your first contribution!! 🚀
Thanks for the contribution.

I saw that you created the bottom property in the theme file, how about using it here?

```suggestion
    padding-bottom: ${dropdown.button.padding.bottom}px;
```